resources/views/sellers/index.blade.php

```diff
@@ -2,87 +2,100 @@
+                        </div>
+                        <div class="col-6">
+                            <p class="mb-0">Last Name</p>
+                            <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">Jones</p>
+                        </div>
                     </div>
-                    <div class="col-6">
-                        <p class="mb-0">Last Name</p>
-                        <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">Jones</p>
+
+                    <div class="col-12">
+                        <p class="mb-0">Username</p>
+                        <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">EmilyJones22</p>
                     </div>
-                </div>
 
-                <div class="col-12">
-                    <p class="mb-0">Username</p>
-                    <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">EmilyJones22</p>
-                </div>
+                    <div class="col-12">
+                        <p class="mb-0">Company Name</p>
+                        <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">NovaLink Global</p>
+                    </div>
 
-                <div class="col-12">
-                    <p class="mb-0">Email</p>
-                    <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">emily-jones58@gmail.com</p>
-                </div>
+                    <div class="col-12">
+                        <p class="mb-0">Email</p>
+                        <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">emily-jones58@gmail.com</p>
+                    </div>
 
-                <div class="col-12">
-                    <p class="mb-0">Phone Number</p>
-                    <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">(603) 555-0123</p>
+                    <div class="col-12">
+                        <p class="mb-0">Phone Number</p>
+                        <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">(603) 555-0123</p>
+                    </div>
                 </div>
-            </div>
 
-            <div class="col-4">
-                <img src="#" alt="" class="rounded-circle picture-circle border border-primary">
+                <div class="col-4">
+                    <img src="#" alt="" class="rounded-circle picture-circle border border-primary">
+                </div>
             </div>
-        </div>
 
-        <div class="row">
-            <div class="col-12">
-                <p class="mb-0">Street Address</p>
-                <p class="rounded py-2 px-3" style="background-color: #E6E6E6;">4140 Parl</p>
-            </div>
+            <div class="row">
+                <div class="col-12">
+                    <p class="mb-0">Street Address</p>
+                    <p class="rounded py-2 px-3" style="background-color: #E6E6E6;">4140 Parl</p>
```

Comment:
I suggest to put the style on a css file and assign a class

Reply:
especially when you're repeatedly using the same style
background-color: #E6E6E6;